app/scripts/app.js

```diff
@@ -227,7 +228,7 @@ angular
       .when('/project/:project/browse/other', {
         templateUrl: 'views/other-resources.html',
         controller: 'OtherResourcesController'
-      })      
+      })
```

Comment:
We should possibly double check and see if everyone has the same settings for eliminating extra white space....